.github/workflows/test_nightlies_on_main.yml

```diff
@@ -37,12 +35,24 @@ jobs:
         with:
           python-version: "3.12"
 
-      - name: Build package
+      - name: Setup build dependencies
+        # PIP_FLAGS: exposed via global env
+        run: |
+          source tools/github/setup_build_env.sh
+
+      - name: Build and install
+        run: |
+          pip install -vv --no-build-isolation .
```

Comment:
```suggestion
          pip install -vv --no-deps --no-build-isolation .
```

Reply:
No, scratch that. `--no-build-isolation` already expects the build dependencies to be present.